Sources/CodeEditCLI/Version.swift

```diff
@@ -16,44 +16,28 @@ extension CodeEditCLI {
         )
 
         func run() throws {
-            // Run an apple script to find CodeEdit.app
-            let pathData = try codeEditURLData()
+            // Print the cli version
+            print("CodeEditCLI: \t\(CLI_VERSION)")
+
+            // File URL of CodeEdit.app
+            let appURL = URL(fileURLWithPath: "/Applications/CodeEdit.app")
```

Comment:
Isn't it better to use the bundle identifier, because not everyone installs in /Applications.

```swift
        guard let appURL = NSWorkspace.shared.urlForApplication(
            withBundleIdentifier: "austincondiff.CodeEdit"
        ) else { return }
```

Reply:
good idea!